goProbe panics when writing out data for an interface without flows

```
ts=2023-03-06T17:04:46Z level=debug caller=capture/capture.go:183 msg="there are currently no flow records available" app_name=goProbe app_version=8e02f0a7 iface=eth1 state=capturing
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x751a17]

goroutine 9 [running]:
github.com/els0r/goProbe/pkg/goDB.dbData({0xc0001a2000?, 0x88?}, 0x0?, 0xc000121830?)
        /usr/src/packages/src/sdwan/security/pkg/OSAGtmi/goProbe/pkg/goDB/db_writer.go:107 +0xb7
github.com/els0r/goProbe/pkg/goDB.(*DBWriter).Write(0xc0002c2930, 0xc000121d60?, {0xc00002d540?}, 0x4?)
        /usr/src/packages/src/sdwan/security/pkg/OSAGtmi/goProbe/pkg/goDB/db_writer.go:57 +0x23d
github.com/els0r/goProbe/pkg/goprobe/writeout.(*Handler).HandleWriteouts.func1()
        /usr/src/packages/src/sdwan/security/pkg/OSAGtmi/goProbe/pkg/goprobe/writeout/handler.go:217 +0x790
created by github.com/els0r/goProbe/pkg/goprobe/writeout.(*Handler).HandleWriteouts
        /usr/src/packages/src/sdwan/security/pkg/OSAGtmi/goProbe/pkg/goprobe/writeout/handler.go:179 +0xaa
```

--- a/pkg/types/hashmap/list.go
+++ b/pkg/types/hashmap/list.go
@@ -17,7 +17,11 @@ type Item struct {
 type List []Item
 
 // Flatten converts a flow map to a flat table / list
-func (a AggFlowMap) Flatten() (v4List List, v6List List) {
+func (a *AggFlowMap) Flatten() (v4List List, v6List List) {
+	if a == nil {
+		return
+	}
+
 	v4List, v6List = make(List, a.V4Map.Len()), make(List, a.V6Map.Len())
 
 	for j, it := 0, a.V4Map.Iter(); it.Next(); j++ {

